FabricExample/e2e/issuesTests/Test2809.e2e.ts

```diff
@@ -0,0 +1,267 @@
+import { device, expect, element, by } from 'detox';
+import { describeIfiOS } from '../e2e-utils';
+
+const expectBackButtonMenuWithTheSameLabel = async (text: string) => {
+   await element(by.text(text)).longPressAndDrag(700, NaN, NaN, element(by.text('VOID')), NaN, NaN, "fast", 0); // open
+   await waitFor(element(by.type('_UIContextMenuView'))).toBeVisible();
+   await expect(element(by.text(text).withAncestor(by.type('_UIContextMenuView')))).toBeVisible();
+   await element(by.text('VOID')).tap(); // close
+   await waitFor(element(by.type('_UIContextMenuView'))).not.toBeVisible();
+   await waitFor(element(by.text(text)).atIndex(1)).not.toExist();
+   await expect(element(by.text(text)).atIndex(0)).toBeVisible();
+};
+
+const expectBackButtonMenuWithDifferentLabels = async (buttonTitle: string, backButtonMenuLabel: string) => {
+   await element(by.text(buttonTitle)).longPressAndDrag(700, NaN, NaN, element(by.text('VOID')), NaN, NaN, "fast", 0); // open
+   await waitFor(element(by.type('_UIContextMenuView'))).toBeVisible();
+   await expect(element(by.text(backButtonMenuLabel).withAncestor(by.type('_UIContextMenuView')))).toBeVisible();
+   await element(by.text('VOID')).tap(); // close
+   await waitFor(element(by.type('_UIContextMenuView'))).not.toBeVisible();
+   await waitFor(element(by.text(backButtonMenuLabel))).not.toBeVisible();
+   await expect(element(by.text(buttonTitle))).toBeVisible();
+};
+
+const expectBackButtonMenuIconAndLabel = async (buttonId: string, backButtonMenuLabel: string) => {
+   await element(by.id(buttonId)).longPressAndDrag(700, NaN, NaN, element(by.text('VOID')), NaN, NaN, "fast", 0); // open
+   await expect(element(by.text(backButtonMenuLabel).withAncestor(by.type('_UIContextMenuView')))).toBeVisible();
+   await waitFor(element(by.type('_UIContextMenuView'))).toBeVisible();
+   await element(by.text('VOID')).tap(); // close
+   await waitFor(element(by.type('_UIContextMenuView'))).not.toBeVisible();
+};
+
+const expectBackButtonMenuToNotExistOnLabel = async (text: string) => {
+   await element(by.text(text)).longPressAndDrag(700, NaN, NaN, element(by.text('VOID')), NaN, NaN, "fast", 0); // open
+   await expect(element(by.type('_UIContextMenuView'))).not.toExist();
+};
+
+
+describeIfiOS('Test2809', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('Test2809 should exist', async () => {
+   await waitFor(element(by.id('root-screen-tests-Test2809')))
+     .toBeVisible()
+     .whileElement(by.id('root-screen-examples-scrollview'))
+     .scroll(600, 'down', NaN, 0.85);
+
+   await expect(element(by.id('root-screen-tests-Test2809'))).toBeVisible();
+   await element(by.id('root-screen-tests-Test2809')).tap();
+ });
+
+   describe('backButtonMenuEnabled: true', () => {
+      describe('backButtonDisplayMode: default', () => {
+         it('with default text', async () => {
```

Comment:
```suggestion
         it('default back title stays visible', async () => {
```
Just one note to all the tests - would be nice if the string in `it` better described our expectations regarding the test, so that it is more readable from terminal what the test does. I'm not claiming that my suggestion is even good, but would be nice to come up with something more readable.

Reply:
changed in: https://github.com/software-mansion/react-native-screens/pull/2848/commits/d1c0127fc0c3b102537b45e840f539d3f7acc3c6